app/views/place/show.html.erb

```diff
@@ -22,26 +24,31 @@
             format: 'service',
             publication_format: 'place',
             postcode: postcode,
-            } %>
+          }.merge(results_anchor ? {
+            action: "/#{params[:slug]}\##{results_anchor}",
+          } : {})
+        %>
       </div>
     </div>
   </section>
   <% if postcode_provided? && !location_error %>
-    <section class="places-results"
-      data-module="auto-track-event"
-      data-track-category="<%= track_category_for_place_results(publication.places) %>"
-      data-track-action="<%= track_action_for_place_results(publication.places) %>"
-      data-track-label="<%= track_label_for_place_results(publication.places) %>">
-      <% if publication.places.any? %>
-        <%= render "govuk_publishing_components/components/heading", {
-          text: "Results #{preposition ||= "near"} <strong>#{postcode}</strong>".html_safe,
-          margin_bottom: 4,
-        } %>
-        <ol id="options" class="govuk-list">
-          <%= render partial: option_partial ||= "option", locals: { places: publication.places } %>
-        </ol>
-      <% end %>
-    </section>
+    <%= content_tag "section", id: results_anchor, class: "places-results",
```

Comment:
Again here, why was this html `section` replaced with a `content_tag`? There is another `section` in this view that hasn't been changed, so now the view is inconsistent.

I think it might be clearer if either: 
1. both sections remain as they are with only the `results_anchor` id being added to this one, or 
2. both sections are changed to a `content_tag` in a separate commit that explains why.

Reply:
I wasn't sure if there was another alternative to writing the conditional attributes of the elements in the template other than this

`<form <% if results_anchor %> id="<%= results_anchor %> <% end %> `

which is why I changed it to content_tag as that handles the case if an attribute is nil a bit more elegantly. Unless the snippet above is OK in which case can revert both instances of `content_tag` to that.

Reply:
Ok, that makes sense. Can you add that to the commit message?

Reply:
I think `[form_with](https://api.rubyonrails.org/v7.0.3/classes/ActionView/Helpers/FormHelper.html#method-i-form_with)` allows an option `id` attribute as well.